code/modules/mob/living/carbon/human/say.dm

```diff
@@ -12,7 +12,7 @@
 		var/obj/item/clothing/mask/chameleon/V = wear_mask
 		if(V.vchange && wear_id)
 			var/obj/item/card/id/idcard = wear_id.GetID()
-			if(istype(idcard))
+			if(istype(idcard) && !idcard.electric)
```

Comment:
this is intended? you can change your voice only with the paper slip?

Reply:
ah wait was adding the not op to everything oops